xclim/indicators/atmos/_precip.py

```diff
@@ -247,6 +250,31 @@ class HrPrecip(Hourly):
     parameters={"tas": {"kind": InputKind.VARIABLE}, "phase": "solid"},
 )
 
+standardized_precipitation_index = PrecipWithIndexing(
+    title="Standardized Precipitation Index (SPI)",
+    identifier="spi",
+    units="",
+    standard_name="spi",
+    long_name="Standardized Precipitation Index (SPI)",
+    description="Precipitations over rolling window {window}-X window, normalized such that SPI averages to 0. The window unit `X` is the minimal time period defined by resampling frequency {freq}",
+    cell_methods="",
+    compute=indices.standardized_precipitation_index,
+    # the freq parameter seems to cause errors when doing the tests
+    parameters=dict(window=1, dist="gamma", method="ML"),
```

Comment:
Doesn't this injects the parameters? Don't we want these params to be accessible to the user?